bpf/lib/lb.h

```diff
@@ -536,7 +535,6 @@ bool lb6_svc_is_localredirect(const struct lb6_service *svc)
 {
 	return svc->flags2 & SVC_FLAG_LOCALREDIRECT;
```

Comment:
```suggestion
  if (!CONFIG(enable_lrp))
     return false;
	return svc->flags2 & SVC_FLAG_LOCALREDIRECT;
```

Or is this implicitly included in the `SVC_FLAG_LOCALREDIRECT` flag?

Reply:
Right now all `lb*_svc_is_localredirect()` calls are guarded by `CONFIG(enable_lrp)` already.